extension/wakewordTraining/wakewordTraining.css

```diff
@@ -0,0 +1,51 @@
+@import url("../options/options.css"); /* temp hack to get some of the options page styling in */
```

Comment:
I'm not sure when we'll want to land the actual specific wakeword support. We'll have to discuss that separately. That would include the trainer.